content/docs/tina-cloud/git-co-authoring.mdx

```diff
@@ -1,51 +1,92 @@
+To solve this, TinaCMS provides two distinct methods for attributing your work: **Act as You** (the recommended approach) or **Co-authoring**.
 
-## What is Co-Authoring?
+***
 
-Co-authoring is a feature that combines the user’s identity with the tina-cloud-app entity in GitHub commit logs. This ensures that when users commit changes to their TinaCMS projects, both their GitHub account and the tina-cloud-app entity are listed as co-authors. When a user with Co-Authoring enabled makes content edits through TinaCMS, the corresponding Git changes are marked as “Co-authored by tina-cloud-app.”
+### Authoring Options at a Glance
 
-![](/tina-cloud-app-good-example.png "✅ Good Example - I can track down Ben Neoh about this change")
+You can choose between two levels of GitHub integration:
 
-## Why do we use Co-Authroing at Tina?
+1. **Act as You (Recommended):** The most seamless and accurate method. TinaCMS will perform all Git actions using **your personal GitHub identity** after you grant authorization.
+2. **Co-authoring:** A simpler, manual option. The `tina-cloud-app` will still make the commit, but it will add your name and email as a **co-author**, attributing the change to you in the commit message.
 
-#### Transparency
+***
 
-* Co-authoring makes it clear that the edits were made with assistance by TinaCMS. This builds trust with the pull request reviewer and other stakeholders, ensuring that changes to the repository were mediated and verified by TinaCMS
-* A co-authored commit also identifies that the specific commit is only a content change - meaning that non-developers will be able to review these commits. If a pull request is only made up of tina-cloud-app commits then it identifies that no javascript/typescript changes were made in this pull request
+## Option 1: Act as You (Full GitHub Authoring)
 
-#### Collaboration Visibility
+This method provides perfect attribution by using your GitHub account to perform all actions. Commits, branches, and pull requests will appear in your repository's history exactly as if you had made them from your own computer.
 
-* Adding TinaCMS as a co-author highlights its role in the content creation process, while also recognizing the author of the change. This makes it useful for tracking which edits were made with TinaCMS, and which were not
-* Enabling Co-Authoring identifies which users in a team are using the TinaCMS tool - ultimately making the GitHub commit history more clear as we can see who is making the commit
+![A commit authored solely by the user, making it clear who made the change.](/img/docs/tina-cloud/github-authoring/github-authoring-commit-example.png "The commit and PR is created by the user, not the bot.")
 
+### How Permissions Are Secured 🔐
 
-#### Context for Pull Requests
+This feature operates on a **dual-permission system**, meaning an action is only successful if **both** the TinaCMS application and **your personal user account** have the required permissions.
```

Comment:
```suggestion
This feature operates on a **dual-permission system**, meaning an action is only successful if **either** **your personal user account** has the required permissions or the TinaCMS application does.
```
Isn't it an either/or model?

We use personal permission by default (when enabled) then fallback to the bot if they don't have access